### PR TITLE
set a default past date time for registered_on so PG triggers work co…

### PR DESCRIPTION
**Pull request by norkans7**
…rrectly

### CODE REVIEW

**Comment by nicpottier** [comment]
Wouldn't setting that to now() be more accurate than a random date in the past? Seems like setting to 2001 will lead to some weird data no?

**Comment by norkans7** [comment]
Our graph shows up to six month in the past I thought using that old date is just fine for this case
using now will assume the contact was just registered so all contact with no date set will appear as if they registered today, Isn't that more confusing?

**Comment by nicpottier** [comment]
We have a created_on on the contact no? Maybe use that? That seems likely to be much closer than either.

**Comment by norkans7** [comment]
Okay

File: ureport/backend/rapidpro.py [hunk 1]
```diff
@@ -202,8 +202,8 @@ def local_kwargs(self, org, remote):
                 registered_on = json_date_to_datetime(registered_on)
 
         if not registered_on:
-            # default to a date in in the past to avoid null in the PG triggers
-            registered_on = json_date_to_datetime('2000-01-01T00:00:00.000')
+            # default to creted_on to avoid null in the PG triggers
```
Comment (nicpottier) [inline ureport/backend/rapidpro.py]: typo

**Pull request merged**